app/src/main/java/au/gov/qld/bdm/documentproduction/document/SignedQRContent.java

```diff
@@ -1,17 +1,31 @@
 package au.gov.qld.bdm.documentproduction.document;
 
+import java.util.Date;
+import java.util.Map;
 import java.util.SortedMap;
+import java.util.TreeMap;
+
+import org.joda.time.LocalDate;
 
 import com.google.gson.Gson;
 
 public class SignedQRContent {
 
-	private SortedMap<String, String> f;
+	private static final String VERSION = "1.1.0";
+
+	private final SortedMap<String, String> f;
+	private final String dId;
```

Comment:
you can use full names here, short forms make this slightly harder to read.

Reply:
you may also want to set json ordering

eg: @JsonPropertyOrder({ "name", "id" }) etc 
via https://www.baeldung.com/jackson-annotations